### DISCUSSION TITLE
Seeing storage used by one user

### DISCUSSION THREAD

**Discussion by demostanis** [community/community#147337] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm curious on how many GBs https://github.com/seanpm2001 uses on your servers with all those repos lol

**Comment by Logophoman** [comment]
Haha surely a ton lol. I mean there's people using GitHub repositories with LFS to store their personal stuff 😉🤣 100.000 commits in a year surely sounds like that but Microsoft doesn't really care i guess...
